api/app/src/main/kotlin/packit/service/PacketService.kt

```diff
@@ -31,7 +34,7 @@ interface PacketService
     fun getPacketsByName(
         name: String
     ): List<Packet>
-
+    fun streamZip(hashes: List<String>, id: String, request: HttpServletRequest, response: HttpServletResponse)
```

Comment:
```suggestion
    fun streamZip(hashes: List<String>, id: String, output: OutputStream)
```

This is all that this function should need as its arguments